aptos-move/framework/aptos-framework/sources/Coin.move

```diff
@@ -264,7 +264,7 @@ module AptosFramework::Coin {
         let supply = &mut borrow_global_mut<CoinInfo<CoinType>>(coin_addr).supply;
         if (Option::is_some(supply)) {
             let supply = Option::borrow_mut(supply);
-            *supply = *supply + amount;
+            *supply = *supply + (amount as u128);
```

Comment:
we should probably check overflow and return proper error code here

Reply:
Done